src/decorators/gb.ts

```diff
@@ -0,0 +1,54 @@
+import IDecorator from "./decorator";
+
+const sixDigitAreaCodes = [
+  "013873",
+  "015242",
+  "015394",
+  "015395",
+  "015396",
+  "016973",
+  "016974",
+  "016977",
+  "016977",
+  "017683",
+  "017684",
+  "017687",
+  "019467",
+];
+
+const decorator: IDecorator = {
+  decorate(phoneNumber: string) {
+    let localNumber;
+
+    if (phoneNumber.substring(2, 4) === "02") {

```

Comment:
Sounds to me like it'd be useful to chuck `phoneNumber.substring(2, 4)` in a constant, given how many times you're checking it in this function. nbd though